src/index.ts

```diff
@@ -48,6 +48,7 @@ export {
   CfnCloudBackupSchedule,
   CfnCloudBackupScheduleProps,
   Export,
+  Link,
```

Comment:
Is Link needed here? I don't see it being used in the changes below?

Reply:
i see that is required for other resources, so i just kept it